.github/workflows/build.yml

```diff
@@ -23,23 +23,28 @@ jobs:
     - name: Install Build Dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y libbtrfs-dev libdevmapper-dev pkg-config build-essential
+        sudo apt-get install -y libbtrfs-dev libdevmapper-dev pkg-config build-essential gcc-aarch64-linux-gnu
 
     - name: Build vib
       run: |
         go get ./...
         go get github.com/ebitengine/purego
-        make build
+        make BINARY_NAME=vib-amd64 build
+        make clean
```

Comment:
the clean would also get rid of the built binary, so it'll have to be copied somewhere before a clean